cypress/utils/commands.js

```diff
@@ -104,9 +104,7 @@ Cypress.Commands.add('bulkUploadDocs', (fixturePath, index) => {
 
 Cypress.Commands.add('importSavedObjects', (fixturePath, overwrite = true) => {
   const sendImportRequest = (ndjson) => {
-    const url = `${Cypress.config().baseUrl}/api/saved_objects/_import?${
```

Comment:
does this work everytime?  for example if the base path is defined for example `_dashboards` for the app is the API also defined `_dashboards` if i remember correctly i think the base path is not added to APIs

Reply:
For now, the net result should be the same given Cypress implicitly prefixes the url with the basePath already ([ref](https://docs.cypress.io/app/core-concepts/best-practices#Setting-a-Global-baseUrl)). Let me know if I'm missing something here though.